core/src/main/scala/io/finch/ToResponse.scala

```diff
@@ -124,14 +124,29 @@ object ToResponse extends HighPriorityToResponseInstances {
     }
   }
 
+  implicit def coproductToResponse[C <: Coproduct, CT <: String](
+    implicit ctr: CoproductToResponse.Aux[C, CT]
+  ): Aux[C, CT] = ctr
+}
+
+trait CoproductToResponse[C <: Coproduct] extends ToResponse[C]
+object CoproductToResponse {
+  type Aux[C <: Coproduct, CT] = CoproductToResponse[C] { type ContentType = CT }
+  def apply[C <: Coproduct](implicit ctp: CoproductToResponse[C]): Aux[C, ctp.ContentType] = ctp
```

Comment:
Should we drop this `apply` method? I don't see it's being used anywhere?

Reply:
Just habit, I will remove it.